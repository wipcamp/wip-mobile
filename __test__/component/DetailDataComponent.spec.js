/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import DetailData from '../../components/DetailDataComponent'

it('rendering', () => {
    const tree = renderer
        .create(
            <DetailData>Test</DetailData>
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})