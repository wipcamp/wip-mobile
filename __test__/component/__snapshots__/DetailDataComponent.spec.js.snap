// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  style={
    Array [
      Object {
        "paddingBottom": 10,
        "paddingTop": 10,
      },
      Object {
        "paddingLeft": 10,
        "paddingRight": 10,
      },
      Object {
        "flexDirection": "row",
      },
      Object {
        "alignItems": "center",
      },
      Object {
        "backgroundColor": "#FFF",
      },
      Object {
        "borderBottomWidth": 0.4,
        "borderColor": "#C8C8C8",
      },
    ]
  }
>
  Test
</View>
`;
